Ical.Net.Tests/CollectionHelpersTests.cs

```diff
@@ -7,7 +7,9 @@
 using System.Collections.Generic;
 using System.Linq;
 using Ical.Net.DataTypes;
+using Ical.Net.Utility;
 using NUnit.Framework;
+using NUnit.Framework.Legacy;
```

Comment:
not in use

Reply:
GitHub warnings The actual and the expected argument is the same 'GetExceptionDates()' would be nice to disappear.

Reply:
not in use
```C#
    private static readonly DateTime _later = _now.AddHours(1);
    private static readonly string _uid = Guid.NewGuid().ToString();

    private static List<RecurrencePattern> GetSimpleRecurrenceList()
        => new List<RecurrencePattern> { new RecurrencePattern(FrequencyType.Daily, 1) { Count = 5 } };
```